src/examples/rest/src/backend/schema/task/entity.ts

```diff
@@ -94,6 +94,9 @@ export class Task extends GraphQLEntity<OrmTask> {
 	@Field(() => Priority, { nullable: true })
 	priority?: Priority;
 
+	@Field(() => [String])
+	test = ['test'];
```

Comment:
Im going to remove this but this is what I used to replicate the error. Does this look correct @kyelewis